app/services/events/list.rb

```diff
@@ -0,0 +1,13 @@
+module Events
+  class List
+    attr_accessor :scope
```

Comment:
Is this used? Could it be worth waiting until we need it to add it? 

YAGNI

Reply:
I'm certain we'll be using it shortly, once we start filtering by event type.